src/PayloadParser.h

```diff
@@ -35,7 +35,7 @@ namespace snowcrash {
     };
 
     /** Request matching regex */
-    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest" SYMBOL_IDENTIFIER "?" MEDIA_TYPE "?[[:blank:]]*";
+    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest([[:blank:]]" SYMBOL_IDENTIFIER ")?" MEDIA_TYPE "?[[:blank:]]*";
```

Comment:
Again though, attributes can contain one or more attribute. As per the specification both are supported. See https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#list-keywords